source/vibe/internal/meta/uda.d

```diff
@@ -102,7 +102,7 @@ template findNextUDA(UDA, alias Symbol, long idx, bool allow_types = false)
 	static assert(idx >= 0, "Index given to findNextUDA can't be negative");
 	static assert(idx <= udaTuple.length, "Index given to findNextUDA is above the number of attribute");
 
-	private template extract(size_t index, list...)
+	public template extract(size_t index, list...)
```

Comment:
Looks like a regression to me. I encountered it in #972 but didn't have time to look into this. I'll see if I can repro / submit an issue tomorrow afternoon.